sentinelsat/sentinel.py

```diff
@@ -110,6 +110,9 @@ class SentinelAPI(object):
     api_url : string, optional
         URL of the DataHub
         defaults to 'https://scihub.copernicus.eu/apihub'
+    page_size : int
+        size of query pages on SciHub
```

Comment:
I'd say this is rather "number of results per query page" as "size" can be ambiguous.